cnames_active.js

```diff
@@ -991,6 +991,7 @@ var cnames_active = {
   "h": "makenowjust.github.io/h.js",
   "h3": "h3rald.github.io/h3",
   "h7ml": "h7ml.github.io/web",
+  "haoxihuan": "haoxihuan.github.io/haoxihuan-js-org",
```

Comment:
```suggestion
  "haoxihuan": "haoxihuan.github.io",
```
It looks like you want to use https://github.com/haoxihuan/haoxihuan.github.io? Not a repository named `haoxihuan-js-org`?

Reply:
Haoxihuan.js.org

Reply:
Haoxihuan.js.org